notebooks/community/pipelines/google_cloud_pipeline_components_bqml_pipeline_anomaly_detection.ipynb

```diff
@@ -0,0 +1,1510 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.        # Train the recostruction model
</pre><p>reconstruction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
done